tensorly/contrib/sparse/backend/numpy_backend.py

```diff
@@ -121,10 +121,10 @@ def partial_svd(self, matrix, n_eigenvecs=None, random_state=None, **kwargs):
         else:
             if n_eigenvecs > min_dim:
                 msg = (
-                    "n_eigenvecs={} if greater than the minimum matrix "
-                    "dimension ({})"
+                    f"{n_eigenvecs = } if greater than the minimum matrix "
+                    f"dimension ({min(matrix.shape)})"
                 )
-                raise ValueError(msg.format(n_eigenvecs, min(matrix.shape)))
+                raise ValueError(msg)
```

Comment:
I agree (also in many other places). I cannot commit this change from GitHub UI because the website says "Applying suggestions on deleted lines is currently not supported."

Reply:
Hmm, odd, I think you'll have to make the change locally and push it to your fork then. That should automatically update this PR.